generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs

```diff
@@ -448,13 +448,21 @@ public class CacheConfiguration<% if (cacheProvider === 'hazelcast') { %> implem
                     .clusterName("infinispan-<%=baseName%>-cluster").globalJmxStatistics()
                     .enabled(jHipsterProperties.getCache().getInfinispan().isStatsEnabled())
                     .allowDuplicateDomains(true).build();
+            } else if (discoveryClient.getInstances(registration.getServiceId()).size() == 0) {
+                return () -> GlobalConfigurationBuilder
+                .defaultClusteredBuilder().defaultCacheName("infinispan-store-cluster-cache").transport()
```

Comment:
missing indent here

Reply:
Done. I wish there was a way these indentations are caught by the IDE. One of the many problems I was thinking about related to EJS. 🤔 🤔